### DISCUSSION TITLE
'None' track in exported .h5 file

### DISCUSSION THREAD

**Discussion by skyshi0** [talmolab/sleap#2007] [Help!] [answered]
Hi all,
I used a multi-animal top-down model to track a single animal. Since I don't need the identity tracking, I set my tracker to 'none'. I'm now trying to use the analysis example code to analyze the exported .h5 file. However, I cannot use "locations = f["tracks"][:].T" to access my data because my track is none. How can I fix this error? Thank you!
Best,
Sky

**Comment by eberrigan** [comment]
Hi @skyshi0,

Sorry, do you want to track your animal or just get frame-by-frame inference of pose?

Best,

Elizabeth

**Reply by skyshi0** [reply]
I want to get a frame-by-frame inference of pose from tracking a single animal, problem was solved after I deleted all tracks. Thank you!

**Comment by roomrys** [comment, marked as answer]
Hello,

I am curious what error message you are getting when you try 
```python
locations = f["tracks"][:].T
```
?

The `"tracks"` key just has a big numpy stored of shape `# frames` by `# nodes` by `2` by `# tracks`. So if you have just a single animal, we expect the numpy to have shape 1 at the end - which is still able to be transposed per the example below:
```python
In [4]: np.random.rand(4,3,2,1).shape
Out[4]: (4, 3, 2, 1)

In [5]: np.random.rand(4,3,2,1).T.shape
Out[5]: (1, 2, 3, 4)
```
.

By the way, if you are missing pose data, this is likely due to accidentally having a track set on a pose somewhere, so it is best to Tracks > Delete Multiple > All for single animal projects before exporting to an analysis file.

Thanks,
Liezl

**Reply by skyshi0** [reply]
Thank you so much! Previously it was (frame count, node counts, 2, 0) but now is (frame count, node counts, 2, 1) after I deleted all tracks.
